.gitignore

```diff
@@ -5,4 +5,4 @@
 /tools/*.pyc
 tags
 tools/sycek/
-tools/xcw/demo/viewer
+tools/xcw/demo/viewer
```

Comment:
Why did you remove the trailing newline?

Reply:
that's probably an accident from switching from the old make build to meson. My original code was written for make and I had to exclude the layout folder